src/Castle.Core/DynamicProxy/Internal/InternalsUtil.cs

```diff
@@ -15,135 +15,43 @@
 namespace Castle.DynamicProxy.Internal
 {
 	using System;
-	using System.Collections.Generic;
-	using System.Linq;
+	using System.ComponentModel;
 	using System.Reflection;
-	using System.Runtime.CompilerServices;
-
-	using Castle.Core.Internal;
-	using Castle.DynamicProxy.Generators.Emitters;
 
 	public static class InternalsUtil
 	{
-		private static readonly IDictionary<Assembly, bool> internalsToDynProxy = new Dictionary<Assembly, bool>();
-		private static readonly Lock internalsToDynProxyLock = Lock.Create();
-
 		/// <summary>
 		///   Determines whether the specified method is internal.
 		/// </summary>
 		/// <param name = "method">The method.</param>
 		/// <returns>
 		///   <c>true</c> if the specified method is internal; otherwise, <c>false</c>.
 		/// </returns>
+		[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Done. I left the `[EditorBrowsable]` attribute, hope you really don't mind. It could in theory be put on the whole class, too, but perhaps you're still going to use that class in the future, so I didn't do that.